data/remnant missions.txt

```diff
@@ -495,3 +560,718 @@ mission "Remnant: Technology Available"
+			`	"Thank you for your help," he says. "Our Inhibitors provide us with something of a mobility advantage against the Korath, but they aren't particularly effective at disabling them. These plasma cannons will open new avenues of research for us." He pauses, then continues. "I also included a bonus for you. It is a 'finder's fee,' as we always appreciate new tech."`
+			`	The weapon technician finishes strapping down the plasma cannons and looks up to see another Remnant approaching. He gives the new Remnant a respectful dip of his head that almost seems like a bow before turning to you and chanting, "Ah, Captain <last>, have you met Tali? She is the engineering prefect here on Viminal. She asked me to let her know when or if you managed to procure the new tech for me."`
+			`	Tali steps forward to face you. "Well met, newcomer among the Remnant," she trills. "Fresh blood amongst us sets songs in motion that have long been in caesura. I have a few things to finish up right now, but I hope to work with you soon." She gives you a polite nod and heads back the way she came.`
+
+
+
+mission "Remnant: Return the Samples"
+	name "Return the Samples"
+	description "A researcher on <planet>, has asked you to return some samples to Nasqueron."
+	source Aventine
+	stopover Nasqueron
+	to offer
+		has "Remnant: Technology Available: offered"
+		random < 50
+	on offer
+		conversation
+			`As you are walking through the spaceport, the researcher you previously assisted with the studies on the void sprites approaches you in a hustle. "We have made some new discoveries!" he exclaims in a trill. "Some of the samples that we took all those years ago are actually eggs of some kind. So the Archon must view our previous research as being disruptive to the void sprites' lifecycles.`
+			`	"We have been storing these eggs in a cryogenic stasis tank, so they should still be viable and intact. Well, most of them, anyway." His notes trail off on a mournful tone.`
+			`	"Could you return some of them to Nasqueron? According to our logs, they were found floating in the lower levels of the atmosphere, where their natural buoyancy should keep them afloat."`
+			choice
+				`	"Sure, I would be glad to return them."`
+				`	"Sorry, I'm really not interested in braving the Archon again."`
+					decline
+
+			`	"Thank you. Since we are continuing to work together, I'll try again." He takes a breath, concentrates, and speaks a short verse about plumage with a complex chord that sounds at once grandiose and self-deprecating. The best you can make out is "Plume." "We can't pay you for this, but if it does appease the Archon we will be able to resume our studies, and who knows where that could lead?"`
```

Comment:
This paragraph is rather confusing to me. I understand from later on that this man's name is Plume, but reading this I wouldn't understand that that is what this paragraph is attempting to convey.

It may also be worth adding a reminder that you need a Gascraft to land on the gas giants and that the Archon will be hostile.